games-strategy/widelands/widelands-0.19.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit eutils versionator cmake-utils
+
+MY_PV=build$(get_version_component_range 2)
+MY_P=${PN}-${MY_PV}-src
+DESCRIPTION="A game similar to Settlers 2"
+HOMEPAGE="https://www.widelands.org/"
+SRC_URI="https://launchpad.net/widelands/${MY_PV}/${MY_PV}/+download/${MY_P}.tar.bz2"
+
+
```

Comment:
let's make repoman happy, it complains about an unnecessary empty line here.

If you use `repoman full` before you commit, or even better, `repoman commit` then these things are checked already locally before you push.